recipes/libsvtav1/all/test_v1_package/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.15)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+add_executable(test_package test_package.cpp)
+target_link_libraries(test_package PUBLIC CONAN_PKG::libsvtav1)
```

```suggestion
add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/../test_package/
                 ${CMAKE_CURRENT_BINARY_DIR}/test_package/)
```
Avoid CONAN_PKG, it will be deprecated soon.